src/ansys/fluent/core/utils/timing_decorator.py

```diff
@@ -0,0 +1,16 @@
+"""Provides a module for timing function decorator."""
```

Comment:
Shall we combine `event_loop.py` and `timing_decorator.py` into a new single `decorators.py` module ? because all of them are decorators.

Reply:
The code in `event_loop.py` is accessing the shared variable, loop. Therefore, only code that needs to access this variable should be in the same module.

Reply:
Ok. We will keep them as separate files. Thank you.